src/assets/styles/base.sass

```diff
@@ -555,4 +559,55 @@ h2:hover .header-link, h3:hover .header-link, h4:hover .header-link, h5:hover .h
   background-color: #1e1f5036
 
 
+div.platform-support
+  background-color: #EDF0F2
```

Comment:
Is there a way to use pre-defined variables for the colors and values here and further down?

Reply:
Yes, I'd like to do that in a separate PR